### DISCUSSION TITLE
[feature] copy file path on info panel

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#14354] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When viewing the file path on disk in the info panel, clicking on the text ("upload/library...") could copy the path to the clipboard for use in terminal/local file storage.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The path is now much easier to copy
